SPRINTLOG.md

```diff
@@ -416,3 +416,11 @@ _Nothing merged in CLI during this sprint_
 # 2024-07-29 - 2024-08-09
 
 - Move raw troubleshooting doc to repo and make small updates ([#1546](https://github.com/ScilifelabDataCentre/dds_web/pull/1546))
+
+# 2024-08-12 - 2024-08-23
```

Comment:
Another SPRINTLOG etiquette question: does this one imply that we're supposed to make a commit even for sprints where we don't edit anything? It feels like that might clutter up the commit history and this file without adding too much info.

Reply:
No not at all, don't make additional commits, just adding this for documentation purposes when a new sprint starts, just so that I know that we didn't forget to add anything here and that it's just that we worked on other stuff / not at all during that specific sprint.